roxctl/generate/netpol/netpol.go

```diff
@@ -13,8 +16,9 @@ type generateNetpolCommand struct {
 	folderPath       string
 	outputFolderPath string
 	outputFilePath   string
-	mergePolicies    bool
-	splitPolicies    bool
+	removeOutputPath bool
+	mergeMode        bool
+	splitMode        bool
```

Comment:
[nit] Maybe `mergeMode` and `splitMode` can be combined into one variable now that they are mutually exclusive. You can have something like.

```golang
type writeMode int
const (
    splitMode writeMode = iota
    mergeMode
)

type generateNetpolCommand struct {
    // [...]
    mode writeMode
}

Reply:
This is a tertiary situation, although I admit I'm not doing the best job of reflecting this in code :)
We have the third option to neither set merge nor split mode. If none are set, `generate netpol` outputs YAML to stdout. 

I could introduce a mode struct and switch depending on that (merge, split, stdout), if that's a bit more intuitive. 
What do you think?

Reply:
You can also add another mode to the const list I mentioned (`stdoutMode`). I'm not sure what would be the added value of defining a struct for the mode but I might be missing something 🤔

Reply:
If we want to take it one step further, we could write it in a way that the function accepts a `writer` interface and we have 3 writers: file, files, stdout

Reply:
After going through the code, I like @lvalerom's idea :) Maybe you could consider applying it? :)